lib/bundler/plugin.rb

```diff
@@ -41,6 +41,20 @@ def install(names, options)
       Bundler.ui.error "Failed to install plugin #{name}: #{e.message}\n  #{e.backtrace.join("\n ")}"
     end
 
+    # Uninstalls the given plugins
+    #
+    # @param [Array<String>] names the name of plugin to be uninstalled
```

Comment:
The list of names is plural? Perhaps `the name of plugin to be uninstalled` -> `the names of plugins to be uninstalled`?